x-pack/solutions/security/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -106,13 +107,13 @@ export const CspPolicyTemplateForm = memo<PackagePolicyReplaceDefineStepExtensio
       }),
       overviewPath: `https://ela.st/cspm-overview`,
       getStartedPath: `https://ela.st/cspm-get-started`,
-      cloudConnectorEnabledVersion: CLOUD_CONNECTOR_VERSION_ENABLED_ESS,
       showCloudTemplates,
       providers: {
         aws: {
           type: CLOUDBEAT_AWS,
           enableOrganization: enableAwsOrganization,
           getStartedPath: `https://www.elastic.co/guide/en/security/current/cspm-get-started.html`,
+          cloudConnectorEnabledVersion: CLOUD_CONNECTOR_PACKAGE_VERSION_ENABLED_AWS,
```

Comment:
Just f.y.i. We would probably be refactor after Reusability Cloud Connector Component to work with `useCloudSetup` and Cloud Connector logic may not live in Cloud Security Posture Package and may live in Fleet.